backend/apps/packages/models.py

```
@@ -0,0 +1,60 @@
+    created_by = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+        related_name="created_packages",
+    )
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        ordering = ["-received_at", "-created_at"]
```

Ruff (0.15.5)

[warning] Mutable default value for class attribute

(RUF012)